docs/whats-new.asciidoc

```diff
@@ -1,189 +1,56 @@
+--
+image::whats-new/images/8.0/data-view.png[]
+--
 [discrete]
-[[sec-7.13-release]]
-== 7.13 release
-
-[discrete]
-[[sec-enhancements-7.13]]
-=== New features
-
-* A new https://github.com/elastic/integrations/tree/master/packages/osquery_manager[Osquery Manager integration] allows you to centrally manage Osquery deployments to Elastic Agents in your Fleet server, and query host data through distributed SQL.
-* A new {fleet} integration allows you to <<download-prebuilt-rules, download the latest version of prebuilt detection rules>> out of the regular {stack} release cycle.
-* 14 new <<prebuilt-rules, prebuilt detection rules>>.
-* A <<fleet-server-upgrade, newly updated {fleet-server} in {kib}>> requires you to redeploy your {agent}s. Review each scenario to ensure you take the appropriate steps to keep your endpoints protected.
-* Add multiple job IDs when <<create-ml-rule, creating a new {ml} rule>>.
-* Updated APIs for <<timeline-api-overview, Timeline>> and <<rules-api-create, Create rule>>.
-* New updates to the Administration page include updated agent statuses and enhancements to <<trusted-apps-ov, Trusted applications>>, including hash validation, improved error messages, and the ability to run a simple search.
-* Alert enhancements:
-** Threat details are now displayed in the <<alerts-ui-manage, Alert details>> flyout. A summarized view of the threat alert is available in the *Summary* tab. Additional threat data is displayed in the *Threat Intel* tab.
-** <<detection-view-and-filter-alerts, Filter alerts>> in the Alerts table by threat presence.
+[[index-updates-8.0]]
+== Index and privilege updates
 
-[discrete]
-[[sec-updates-7.13]]
-=== New requirements
+*Index changes*
 
-* To use {es-sec}, at least one node in an {es} cluster <<sec-requirements, needs the `transform` role>>.
+* The system index for detection alerts has been renamed from `.siem-signals` to `.alerts-security.alerts`. As such, the schema used for alert documents in {elastic-sec} has changed. Users that access documents in the `.siem-signals` indices will be able to modify their API queries and scripts to operate properly on the new 8.0 alert documents.
+* To view alerts, users need manage, `write`, `read`, and `view_index_metadata` privileges to two new indices, `.alerts-security.alerts` and `.internal.alerts-security.alerts`. Existing users who are upgrading to 8.0 can retain their privileges to the `.siem-signals` index. Refer to <<detections-permissions-section>> for more information.
+* To <<preview-rules, preview rules>>, users need `read` access to the new `.preview.alerts-security.alerts` index. Refer to <<detections-permissions-section>> for more information.
```

Comment:
Should this be preceded by a section header that's similar to what's on line 26?